project.md

```diff
@@ -58,11 +76,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+	External DSLs are generally all going to be heavy on implementing semantics, but I still think the language has a lot of design decisions that will end up making it worthwhile as a project.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+	It's a relatively large project. One big issue with it is that it's easy to slowly add on more and more features. The looser I am with the limits around what to design, the more work the project could grow into, so it's important to go in with a specific plan for this project.
```

Comment:
Another way to restrict the domain of this project is to maybe specify "I only want to focus on music with x time signature and in the C major". I think you're absolutely right that you should constrict the scope of your project so that feature creeping doesn't befall it.